src/BrowsershotLambda.php

```diff
@@ -54,4 +56,32 @@ protected function throwError(SettledResult $response): void
 
         $response->throw();
     }
+
+    /**
+     * @throws CouldNotTakeBrowsershot
+     */
+    public function saveToS3(string $targetPath, string $disk = 's3')
+    {
+        $this->setOption('s3', [
+            'path' => $targetPath,
+            'region' => config('sidecar.region'),
+            'bucket' => config("filesystems.disks.$disk.bucket")
+        ]);
```

Comment:
I suggest making this configurable for the user in case they want to specify another bucket or another region.